clifford/_multivector.py

```diff
@@ -739,6 +743,50 @@ def normalInv(self, check=True) -> 'MultiVector':
         """
         return self._pick_inv(fallback=False if check else None)
 
+    def hitzer_inverse(self):
+        """
+        Performs the inversion operation as described in the paper :cite:`Hitzer_Sangwine_2017`
+        """
+        tot = sum(self.layout.sig != 0)
+        if tot <= 5:
+            if tot == 1:
+                # Equation 4.3
+                mv_invol = self.gradeInvol()
+                numerator = mv_invol
+                denominator = (self * mv_invol)[()]
+            elif tot == 2:
+                # Equation 5.5
+                mv_conj = self.conjugate()
+                numerator = mv_conj
+                denominator = (self * mv_conj)[()]
+            elif tot == 3:
+                # Equation 6.5
+                mv_reverse = ~self
+                mv_invol = self.gradeInvol()
+                mv_conj = self.conjugate()
+                numerator = (mv_conj * mv_invol * mv_reverse)
+                denominator = (self * numerator)[()]
+            elif tot == 4:
+                # Equation 7.8
+                mv_conj = self.conjugate()
+                conj_mul_mv = mv_conj * self
+                numerator = (conj_mul_mv - 2 * (conj_mul_mv)(3, 4)) * mv_conj
+                denominator = (numerator * self)[()]
```

Comment:
![image](https://user-images.githubusercontent.com/425260/86133584-677e6a00-bae0-11ea-9eec-0fbe7accaef9.png)

```suggestion
                # Equation 7.7
                mv_conj = self.conjugate()
                conj_mul_mv = self * mv_conj
                numerator = mv_conj * (conj_mul_mv - 2 * (conj_mul_mv)(3, 4))
                denominator = (self * numerator)[()]
```